README.md

```diff
@@ -1,158 +1,552 @@
+2. Install Dojo, it is a self-contained binary, so just place it somewhere on the `PATH`.
+On **Linux**:
+```bash
+DOJO_VERSION=0.4.0
+wget -O dojo https://github.com/ai-traders/dojo/releases/download/${DOJO_VERSION}/dojo_linux_amd64
+sudo mv dojo /usr/local/bin
+sudo chmod +x /usr/local/bin/dojo
+```
+
+Done. Now you have sufficient environment to build any project which leverages dojo images.
+
+Let's build this [java project](https://github.com/tomzo/gocd-yaml-config-plugin) using dojo:
+
+```bash
+git clone https://github.com/tomzo/gocd-yaml-config-plugin.git
+cd gocd-yaml-config-plugin
+dojo "gradle test jar"
+```
+
+The output should start with a docker command that was executed:
+```console
+tomzo@073c1c477b1f:/tmp/gocd-yaml-config-plugin$ dojo "gradle test jar"
+2019/04/28 14:01:40 [ 1]  INFO: (main.main) Dojo version 0.3.2
+2019/04/28 14:01:40 [17]  INFO: (main.DockerDriver.HandleRun) docker command will be:
+ docker run --rm -v /tmp/gocd-yaml-config-plugin:/dojo/work -v /home/tomzo:/dojo/identity:ro --env-file=/tmp/dojo-environment-dojo-gocd-yaml-config-plugin-2019-04-28_14-01-40-89074376 -v /tmp/.X11-unix:/tmp/.X11-unix -ti --name=dojo-gocd-yaml-config-plugin-2019-04-28_14-01-40-89074376 kudulab/openjdk-dojo:1.0.1 "gradle test jar"
+Unable to find image 'kudulab/openjdk-dojo:1.0.1' locally
+1.0.1: Pulling from kudulab/openjdk-dojo
+```
+Then you should see `docker pull` output and after creating the container, all tests being executed.
+The build artifacts land in `build/libs/`, because that is how gradle behaves. These artifacts are available on our docker host.
+Things to notice:
+ * We have pulled [`kudulab/openjdk-dojo` docker image](https://github.com/kudulab/docker-openjdk-dojo) and created container from it.
+ * Current directory `/tmp/gocd-yaml-config-plugin` was [mounted](https://docs.docker.com/storage/volumes/) to `/dojo/work`
+ * Home directory on host `/home/tomzo` was [mounted](https://docs.docker.com/storage/volumes/) as readonly to `/dojo/identity`
+ * After `gradle test jar` has finished, the docker container has exited and was removed. This is because we provided a command to `dojo`.
+
+Every dojo image supports interactive mode. In above example, we can also run `dojo` at the root of the project to get dropped into an interactive shell.
+Then we can work in the container for longer time, very much like in a [vagrant VM](#vs-vagrant).
+Thanks to the mounted directory from the host, we can work on the project files using any other tools on our host, while container with java tools shares the same files.
+
+This is a quickstart preview to give you a sense of how you would work with dojo. A more complete guide is being developed in this [readme](#table-of-contents) and an upcoming documentation site.
+
+# Installation
+
+### Prerequisites
+
+You must be able to run a local docker daemon that can execute linux docker containers.
+In practice this means Dojo works on **Linux or Mac**.
+Tests are executed only on Linux.
```

Comment:
This sentence is not very clear to me. Maybe we could write instead: "Dojo is continuously tested only on Linux."